core/domain/auth_services_test.py

```diff
@@ -237,59 +236,43 @@ def test_delete_association_when_it_is_missing_does_not_raise(self) -> None:
         # Should not raise.
         auth_services.delete_external_auth_associations('does_not_exist')
 
-    def test_auth_session_established_or_destoryed(self) -> None:
-        auth_section = []
-        def mock_establish_auth_session(
-            _: webapp2.Request,
-            __: webapp2.Response
-        ) -> None:
-            auth_section.append('established')
-
-        def mock_destroy_auth_session(
-            _: webapp2.Response
-        ) -> None:
-            auth_section.remove('established')
-
-        with self.swap(
-            platform_auth_services,
-            'establish_auth_session',
-            mock_establish_auth_session
-        ):
-            auth_services.establish_auth_session(
-                webapp2.Request.blank('/'),
-                webapp2.Response()
+    @mock.patch(
+            'core.domain.auth_services.platform_auth_services.'
+            'establish_auth_session'
             )
-            self.assertEqual(['established'], auth_section)
-        with self.swap(
-            platform_auth_services,
-            'destroy_auth_session',
-            mock_destroy_auth_session
-        ):
-            auth_services.destroy_auth_session(webapp2.Response())
-            self.assertEqual([], auth_section)
-
-    def test_super_admin_granted_or_revoked(self) -> None:
-        super_admin_privilage = []
-        def mock_grant_super_admin_privileges(uid: str) -> None:
-            super_admin_privilage.append(uid)
-
-        def mock_revoke_super_admin_privileges(uid: str) -> None:
-            super_admin_privilage.remove(uid)
-
-        with self.swap(
-            platform_auth_services,
-            'grant_super_admin_privileges',
-            mock_grant_super_admin_privileges
-        ):
-            auth_services.grant_super_admin_privileges('uid1')
-            self.assertEqual(['uid1'], super_admin_privilage)
-        with self.swap(
-            platform_auth_services,
-            'revoke_super_admin_privileges',
-            mock_revoke_super_admin_privileges
-        ):
-            auth_services.revoke_super_admin_privileges('uid1')
-            self.assertEqual([], super_admin_privilage)
+    @mock.patch(
+        'core.domain.auth_services.platform_auth_services.destroy_auth_session'
+    )
+    def test_auth_session_established_or_destroyed(
+        self, mock_destroy_auth_session: mock.Mock,
+        mock_establish_auth_session: mock.Mock) -> None:
```

Comment:
Move ") -> None" to the next line and deindent by 4.

Please do similarly throughout your PR so it's easy to distinguish the function signature from the function body.